lib/components/NumberInput/NumberInput.tsx

```diff
@@ -2,6 +2,8 @@ import React, { memo } from 'react';
 
 import { withEnhancedInput } from '../InputBase';
 
+const isEdge =
+	navigator && navigator.userAgent && /edge/gi.test(navigator.userAgent);
```

Comment:
```suggestion
	navigator && navigator.userAgent && /edge/i.test(navigator.userAgent);
```

The `/g` isn't required. It doesn't affect performance, but is vastly simpler to read.

Reply:
also; you shouldn't need to `defined` check the `navigator.userAgent` property, afaik that _should_ always exist. Perhaps SSR support?